model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -483,7 +487,7 @@ def _create_next_model_run_config(
         composing_model_config_variants: List[ModelConfigVariant],
     ) -> ModelRunConfig:
         perf_analyzer_config = self._get_next_perf_analyzer_config(
-            model_config_variant.variant_name, model, model_index
+            model.model_name(), model, model_index
```

Comment:
Was this an intended change? Just wanted to make sure since everything seemed like it was changing to use `model_config_variant.variant_name`.